src/main/scala/dsptools/Chisel3TestersCompatibility.scala

```diff
@@ -0,0 +1,14 @@
+package chisel3.iotesters
```

Comment:
Also, why do you need to be the ones providing guarantees that "hacks" keep working? My opinion is:

Make a list of public functions that you know are correct and always will support.
Keep other stuff public as well to make library writing easier for people, but it's up to the library or w/e folks to make sure that their stuff doesn't break as Chisel changes. They have their own API, which should be even more front-facing if their libraries are being used, but that's not your concern.